docs/auth/tokens.rst

```diff
@@ -101,7 +101,7 @@ Field details:
     :Access mode: read, write
     :Type: string (time duration: ``[DD] [HH:[MM:]]ss[.uuuuuu]``) or ``null``
 
-    Maximum token age.  If ``created + max_age`` is less than the current time,
+    Maximum token age.  If ``created + max_age`` is greater than the current time,
     the token is invalidated.  Invalidated tokens are not automatically deleted
```

Comment:
The expiration time of a token is its creation date plus the maximum age. The token is invalid if the expiration date is in the past, that is, if it is *less* than the current time.